### PR TITLE
Create eip155-10522.json

### PR DESCRIPTION
**Pull request by mylandpay**
c.t.eip155-10522.json
ths

### CODE REVIEW

File: _data/chains/eip155-10522.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://docs.mylandtec.com",
+  "shortName": "XMT",
+  "chainId": 10522,
+  "networkId": 10522,
+  "slip44": 700,
+  "explorers": [{
+    "name": "xmltecscan",
+    "url": "https://e.mylandtec.com",
```
Comment (ligi) [inline _data/chains/eip155-10522.json:21]: I cannot reach this site

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/chains/eip155-10522.json (hunk 1 above)
Comment (mylandpay) [reply _data/chains/eip155-10522.json:21]: Thanks for your reminding. I will improve it in the near future.
Thank you.^..^

**Review by JurbinM — APPROVED** [review]

File: _data/chains/eip155-10522.json (hunk 1 above)
Comment (Jay4577) [reply _data/chains/eip155-10522.json:21]: > I cannot reach this site

I need help plz

Comment (Jay4577) [reply _data/chains/eip155-10522.json:21]: I need help

**Pull request closed without merging**